src/app/[locale]/_component/noProfile/noProfile.tsx

```diff
@@ -21,7 +22,7 @@ export const NoProfile = (): React.ReactElement => {
   }, []);
 
   const handleBtnContinue = () => {
-    pushWithLocale(process.env.NEXT_PUBLIC_URL_IO || '');
+    pushWithLocale(ROUTES.LOGIN);
```

Comment:
This is not explained in the PR description.

Reply:
After the change, I had to adjust the pages that used the absolute path by using the 'pushwithlocale' function and the relative path